root/static/scripts/work.js

```diff
@@ -189,3 +189,19 @@ store.subscribe(renderWorkLanguages);
 renderWorkLanguages();
 
 MB.Control.initializeBubble('#iswcs-bubble', 'input[name=edit-work\\.iswcs\\.0]');
+
+let $typeBubble = $('#type-bubble')[0];
+let typeIdField = 'select[name=edit-work\\.type_id]';
```

Comment:
This variable is used only once. Please move its value directly to the function call two lines below.

Reply:
Actually, it's used again after that to bind to its `change` event